lib/iris/plot.py

```diff
@@ -845,7 +845,9 @@ def _replace_axes_with_cartopy_axes(cartopy_proj):
                 ylabel=ax.get_ylabel(),
             )
         else:
+            position = ax.get_position(original=True)
             _ = fig.add_axes(
+                position,
                 projection=cartopy_proj,
                 title=ax.get_title(),
                 xlabel=ax.get_xlabel(),
```

Comment:
I find it odd that these arguments aren't exhaustive, and that cartopy doesn't offer a `GeoAxes.from_mpl_axes` method, but that's not relevant for this PR I don't think. Though the original issue mentions that title font isn't preserved by this currently either.

Reply:
I've no idea why these specific attributes were chosen.  I think this function dates back to the very early days of Iris.  In my own use-cases, I think I always call `iplt`/`qplt` functions before doing any titles/labels anyway, so it's only really the position/gridspec that would matter.  Of course I may not be representative of the average user!

> cartopy doesn't offer a GeoAxes.from_mpl_axes method

I did raise a feature request in cartopy for that, but it seems it's not a goer https://github.com/SciTools/cartopy/issues/1756

Reply:
Issue raised (#4664)